tests/dnp3-dnp3_data-alert/test.yaml

```diff
@@ -3,8 +3,39 @@ requires:
     - HAVE_LIBJANSSON
 
 checks:
+  - filter:
+      count: 4
+      match:
+        event_type: alert
+        alert.signature_id: 1
+  - filter:
+      count: 4
+      match:
+        event_type: alert
+        alert.signature_id: 2
+  - filter:
+      count: 0
+      match:
+        event_type: alert
+        alert.signature_id: 3
   - filter:
       count: 4
       match:
         event_type: alert
         alert.signature_id: 4
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 5
+        pcap_cnt: 17
+  - filter:
+      count: 0
+      match:
+        event_type: alert
+        alert.signature_id: 6
```

Comment:
In `DetectDNP3DataTest02`, I see that we expect the signature that is close to our sid:6 here to match. So I think we should change that signature to have a content that would match, maybe, then add a check for the pcap_cnt, like you did for sid:7.

Reply:
Sid:6 is not expected to match based on the flow and the content. only sid:5 is expected to match

Reply:
You have it sorted out, sorry for the noise :P